src/NuGet.Core/NuGet.Protocol.Core.Types/Resources/IPackageSearchResource.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace NuGet.Protocol.Core.Types
+{
+    public interface IPackageSearchResource : INuGetResource
```

Comment:
I believe it was done originally to keep resources from being multiple types. An interface should be fine here, it's just a little inconsistent with the rest.